composer.lock

```diff
@@ -650,7 +652,7 @@
                     "homepage": "https://nette.org/contributors"
                 }
             ],
-            "description": "💎 Nette Dependency Injection Container: Flexible, compiled and full-featured DIC with perfectly usable autowiring and support for all new PHP 7.1 features.",
+            "description": "? Nette Dependency Injection Container: Flexible, compiled and full-featured DIC with perfectly usable autowiring and support for all new PHP 7.1 features.",
```

Comment:
This looks like a composer bug on Windows.

Reply:
Did you run the update on Windows? 😱

Reply:
Yes. 😹 Composer should be portable, right? 😅

Reply:
Apparently, it's not only a Windows issue. I'm getting the same consistently on one of my two Linux machines (see #3604). Did you have a chance to look deeper or maybe report an issue? I haven't found anything related.